benchmarks/cutlass_benchmarks/weight_shapes.py

```diff
@@ -0,0 +1,74 @@
+WEIGHT_SHAPES = {
+    "mistralai/Mistral-7B-v0.1/TP1": [
```

Comment:
A better way to format these shapes would be just having shapes for TP1. Then we could accept `--tp-size` in `cutlass_benchmark.py` to calculate the desired shape.

Reply:
I see, but then we need to store information on which dimension to divide. i.e.
```
    "mistralai/Mistral-7B-v0.1/TP1": [
        [4096, 6144],
        [4096, 4096],
        [4096, 28672],
        [14336, 4096],
    ],
    "mistralai/Mistral-7B-v0.1/TP2": [
        [4096, 3072],  # divide dim 2 
        [2048, 4096],  # divide dim 1
        [4096, 14336], # divide dim 2
        [7168, 4096],   # divide dim 1
    ],
```
perhaps we can store the divide dim as a third element in the list ?

Reply:
That sounds good to me :)

Reply:
yeah. definitely better 👍